apps/test/integration/levelSolutions/gamelab/collision_blocks.js

```diff
@@ -302,9 +302,10 @@ module.exports = {
       },
       function validateResult(assert) {
         var debugOutput = document.getElementById('debug-output');
-        assert.equal(
-          debugOutput.textContent,
-          '"in callback"' + '"after callback"' + '"done"'
+        assert(
+          debugOutput.textContent.includes(
+            '"in callback"' + '"after callback"' + '"done"'
```

Comment:
Since the goal of this test is to prove that the `overlap()` callback runs synchronously, I don't feel bad weakening this assertion to ignore any extra `done`s recorded in the debug output, as long as the sequence is present.